xsnap/sources/xsnap.c

```diff
@@ -6,13 +6,13 @@ static void xsBuildAgent(xsMachine* the);
 static void xsPlayTest(xsMachine* the);
 static void xsPrintUsage();
 
-static void xs_clearTimer(xsMachine* the);
+//static void xs_clearTimer(xsMachine* the);
```

Comment:
@kriskowal maybe we don't want to turn these timer functions off after all.

Reply:
Removed and adjusted the callback tables all they way from here to the HEAD of the stack.